src/features/faq.feature

```diff
@@ -0,0 +1,176 @@
+Feature: Tests for FAQ page
+
+    Scenario: Check FAQ page title and description
+    Given the user is on the Faq page
+    Then the Faq title, header and description should be correct
```

Comment:
Could be three steps, one for title, one for header and one for description

Reply:
Test was rewritten, more steps added.